<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTableView" name="importView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>I&amp;mport to this category:</string>
       </property>
       <property name="buddy">
        <cstring>categoryComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>&amp;Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>&amp;Open File..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="removeFromImportListAction">
   <property name="text">
    <string>Remove From Import List</string>
   </property>
   <property name="toolTip">
    <string>Removes entry from the import list</string>
   </property>
  </action>
  <action name="addToImportListAction">
   <property name="text">
    <string>Add To Import List</string>
   </property>
   <property name="toolTip">
    <string>Adds entry to the import list</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>openFileButton</tabstop>
  <tabstop>categoryComboBox</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>importView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
